boards/ti/am243x_evm/am243x_evm_am2434_r5f0_0.dts

```diff
@@ -0,0 +1,84 @@
+/* Copyright (c) 2025 Texas Instruments Incorporated
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <zephyr/dt-bindings/gpio/gpio.h>
+#include <ti/am64x_r5.dtsi>
+#include "am243x_evm_am2434_r5f0_0-pinctrl.dtsi"
+
+/delete-node/ &mbox4;
+/delete-node/ &mbox5;
+/delete-node/ &mbox7;
```

Comment:
These should be disabled already, do you really need to delete them also?

Reply:
you are correct but I just deleted them since they cannot be used at all by r5fss0_0